File: ARTPARK/ArUco/openCamera.cpp
#include <opencv2/opencv.hpp>
#include <opencv2/aruco.hpp>

int main() {

    cv::VideoCapture cap(0);

    while (true)
    {
        cv::Mat frame;
        cap >> frame;

        cv::imshow("Camera", frame);

        if (cv::waitKey(1) == 27) {
            break;
        }

    }
    
    cap.release();
    cv::destroyAllWindows();

    return 0;
}

File: ARTPARK/ArUco/openCamera.py
import cv2

cap = cv2.VideoCapture(0)

while cap.isOpened():

    ret, image = cap.read()
 
    cv2.imshow('image',image)

    key = cv2.waitKey(1)
    if key == 27:
        break

cap.release()
cv2.destroyAllWindows()
